Documentation/network/clustermesh/services.rst

```diff
@@ -62,6 +62,49 @@ Below example will expose remote endpoint without sharing local endpoints.
      selector:
        name: rebel-base
 
+Synchronizing Kubernetes EndpointSlice (Beta)
+#############################################
+
+.. include:: ../../beta.rst
+
+By default Kubernetes EndpointSlice synchronization is disabled on Global services.
+To have Cilium discover remote clusters endpoints of a Global Service
+from DNS or any third party controllers, enable synchronization by adding
+the annotation ``service.cilium.io/global-sync-endpoint-slices: "true"``.
```

Comment:
I will not block the PR on this. @MrFreezeex can we have a follow up PR with the followin sentence:
```
the annotation ``service.cilium.io/global-sync-endpoint-slices: "true"``.
This will allow Cilium to create k8s endpoint slices that belong a remote cluster for services that have that annotation.
```

Reply:
I will try to include that change in a separate commit in one of my followup PRs that I intend to do this about this :D